test/.eslintrc

```diff
@@ -2,4 +2,7 @@ extends:
   - "../.eslintrc"
 
 env:
-  mocha: true
+  jest: true
+
+rules:
+  implicit-dependencies/no-implicit: 0
```

Comment:
What's the reasoning here?

Reply:
enzyme is a devdep

Reply:
Can we configure it to either a) error but include dev deps, or b) warn. I'm not super happy with just turning it off altogether.

Reply:
Oh wait, this is in `./test` - error but include dev deps is the right answer then.